src/Sulu/Bundle/CategoryBundle/Twig/CategoryTwigExtension.php

```diff
@@ -83,12 +93,21 @@ public function getCategoriesFunction($locale, $parentKey = null)
             'sulu_categories',
             \func_get_args(),
             function($locale, $parentKey = null) {
-                $entities = $this->categoryManager->findChildrenByParentKey($parentKey);
-                $categories = $this->categoryManager->getApiObjects($entities, $locale);
-                $context = SerializationContext::create();
-                $context->setSerializeNull(true);
-
-                return $this->serializer->serialize($categories, $context);
+                try {
+                    $entities = $this->categoryManager->findChildrenByParentKey($parentKey);
+                    $categories = $this->categoryManager->getApiObjects($entities, $locale);
+                    $context = SerializationContext::create();
+                    $context->setSerializeNull(true);
+
+                    return $this->serializer->serialize($categories, $context);
+                } catch (CategoryKeyNotFoundException $exception) {
+                    $this->logger->error(
```

Comment:
We are not logging errors in other cases when something don't exist when loaded over twig extension. So I would not do it here for consistents.

Reply:
OK.